README.md

```diff
@@ -59,10 +59,10 @@ SupportKit.init({appToken: 'your_app_token'});
 ### Individual functions
 
 #### init(options)
-Initializes the SupportKit widget in the web page using the specified options
+Initializes the SupportKit widget in the web page using the specified options. It returns a promise that will resolve when the widget is ready if you need to do different things at different times.
```

Comment:
Better phrasing? It allows you to keep a reference where you can always be sure the widget is init-ed